src/executors/ExecutorServiceEx.java

```diff
@@ -0,0 +1,70 @@
+package executors;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class ExecutorServiceEx {
+    SimpleDateFormat sdf = null;
+    private final int COUNT = 5;
+
+    ExecutorServiceEx() {
```

Comment:
add access modificator

Reply:
and it is not a good decision to write all logic in constructor
use main() for it